packages/docs/docs/guides/rendering.md

```diff
@@ -1,74 +1,179 @@
 # Rendering
 
-Once you assemble your AI.JSX component tree, you'll want to render it into text or UI components. The way you do it depends on how you're using AI.JSX.
+Once you assemble your AI.JSX component tree, you'll want to render it into text or UI components. The way in which components get assembled depends on your scenario and how you are using AI.JSX. By default (and to provide the best end-user experience) AI.JSX streams results from all calls you make to an LLM.
 
-## From clientside React
+## Streaming in AI.JSX
 
-:::note Architectures
-This applies to the following architectures:
+Rendering an AI.JSX program yields a stream. By default, AI.JSX will give you partial results for the output. Alternatively, you can choose to use append-only streams which outputs sequential chunks that can then be concatenated together.
```

Comment:
I don't think this sentence will make sense to low-context people. Maybe what I wrote here is better, but I'm not sure. :smile:

```suggestion
Rendering an AI.JSX program yields a stream. You can choose to consume it one of two ways:
1. A stream of deltas (that you have to `+=` together to get the final result)
1. A stream of in-progress versions of the final full result.

We'll explore these two options in more detail below.
```

Reply:
cool thanks. took your suggestion here and reworked it a bit.